static/app/views/performance/browser/resources/utils/useResourcesQuery.ts

```diff
@@ -53,10 +54,7 @@ export const useResourcesQuery = ({sort}: {sort: ValidSort}) => {
     'count()': row['count()'] as number,
     'spm()': row['spm()'] as number,
     'span.group': row['span.group'].toString(),
-    'resource.render_blocking_status': row['resource.render_blocking_status'] as
-      | ''
-      | 'non-blocking'
-      | 'blocking',
+    'resource.render_blocking_status': row['resource.render_blocking_status'] as string,
```

Comment:
why the change?

Reply:
Ooops, good call, that was not intentional